docs/UnInstall_IML.md

```diff
@@ -0,0 +1,184 @@
+[**Table of Contents**](index.md)
+
+# Uninstalling IML
+
+![clustre](md_Graphics/uninstall_sm.jpg)
+
+# **Step 1**: Uninstall the IML Manager
+
+### Stop the manager and httpd 
```

Comment:
We don't use httpd anymore.

Should be nginx.

Reply:
changed